docs/quick_tutorial/static_assets.rst

```diff
@@ -43,13 +43,18 @@ Steps
    .. literalinclude:: static_assets/tutorial/static/app.css
     :language: css
 
-#. Make sure we haven't broken any existing code by running the tests:
+#. Add a functional test that requires the newly added static file to be delivered:
+
+   .. literalinclude:: static_assets/tutorial/tests.py
+    :linenos:
```

Comment:
Now only the added lines are included in this tutorial page: larsblumberg@7b3dd65

I'm happy to volunteer updating all quick tutorials steps. We however want to keep _this_ PR focussed on one single change (adding a test case for the added static file).